BrainPortal/app/views/tools/_tools_table.html.erb

```diff
@@ -112,7 +112,8 @@
     end
 
     t.column("Help", :help) do |tool|
-      path = tool.cbrain_task_class.constantize.public_path("edit_params_help.html") rescue nil
+      path   = tool.cbrain_task_class.constantize.public_path("edit_params_help.html") rescue nil
+      path ||= tool.cbrain_task_class.constantize.help_filepath rescue nil # Boutiques help-page
```

Comment:
Where would the help_filepath come from or point to? It can only be part of static assets for the site, right? Not sure I get this change.